api/profiles/profiles.proto

```diff
@@ -0,0 +1,87 @@
+syntax = "proto3";
```

Comment:
For the record, if we want a nice documentation of these using `buf doc` this will need to be commented nicely. All fields with appropriate comments, otherwise they just show up blank.

Reply:
that's true, wego core's `Applications` and enterprise's `ClustersService` ([here](https://github.com/weaveworks/weave-gitops-enterprise/blob/main/cmd/clusters-service/api/capi_server.proto)) services are also missing doc comments. we could open a separate issue for this :) also, couldn't find `buf doc`, are you referring to `proto-gen-doc`? 🤔

Reply:
We have pretty good coverage on doc coverage in `Applications`: 
https://github.com/weaveworks/weave-gitops/blob/60848106b264b6b56b1733d8949fc919966336ac/api/applications/applications.proto#L180

Reply:
you're right, we'll add some doc comments for profiles! 👍